script.js

```diff
@@ -0,0 +1,200 @@
+const resume = {
```

Comment:
Might be just reaching to add a comment here, but we could also consider throwing this into a txt/JSON file and using 'import' to fetch the data.